.circleci/config.yml

```
@@ -237,7 +237,7 @@ jobs:
                   tag=$corresponding_milestone_title
                   change_notes=$(./gradlew getChangelog --no-header -q)
                   hub release create \
-                    --attach build/distributions/git-machete-intellij-plugin-*.zip \
+                    --attach $(./gradlew -q printPluginZipPath) \
```

To be extra safe for future & avoid shell parsing surprises:
```suggestion
                    --attach "$(./gradlew -q printPluginZipPath)" \
```